cdk/stack/waf.py

```diff
@@ -0,0 +1,73 @@
+from aws_cdk import (
+    CfnOutput,
+    Stack,
+)
+from aws_cdk import aws_wafv2 as wafv2
+from constructs import Construct
+
+
+class WafStack(Stack):
+    def __init__(self, scope: Construct, id: str, **kwargs):
+        super().__init__(scope, id, **kwargs)
+
+        web_acl = wafv2.CfnWebACL(
+            self,
+            "SimpleBotProtectionWAF",
+            default_action=wafv2.CfnWebACL.DefaultActionProperty(allow={}),
+            scope="CLOUDFRONT",
+            visibility_config=wafv2.CfnWebACL.VisibilityConfigProperty(
+                cloud_watch_metrics_enabled=True,
+                metric_name="SimpleBotProtectionWAF",
+                sampled_requests_enabled=True,
+            ),
+            rules=[
+                # 1️⃣ Rate limiting rule: 1000 requests per 5 min per IP
+                wafv2.CfnWebACL.RuleProperty(
+                    name="RateLimitRule",
+                    priority=1,
+                    action=wafv2.CfnWebACL.RuleActionProperty(block={}),
+                    statement=wafv2.CfnWebACL.StatementProperty(
+                        rate_based_statement=wafv2.CfnWebACL.RateBasedStatementProperty(
+                            limit=1000,
```

Comment:
I reckon we go lower here.
When we were basically under attack I set 100 requests per minute and that didn't block anything legit, so how about 500 per 5 mins